integration_tests/tests/vault/set_admin.rs

```diff
@@ -74,6 +74,23 @@ mod tests {
         assert_eq!(vault.admin, vault_admin.pubkey());
 
         let new_admin = Keypair::new();
+        {
```

Comment:
can you throw this into a new test to keep this one simpler?

Reply:
Yeah, I added new test case `test_update_secondary_admin`